setting_up_firely_server/configuration/environment_variables.rst

```diff
@@ -25,7 +25,7 @@ The format for the environment variables is:
 
     VONK_<setting_level_1>[:<setting_level_n>]*
 
-So you start the variable name with the prefix 'VONK\_', and then follow the properties in the json settings, separating each level with a colon ':'. Some examples:
+So you start the variable name with the prefix ``VONK_``, and then follow the properties in the json settings, separating each level with a colon ``:``. Some examples:
```

Comment:
Not you code, but we should replace the `:` separator by the `__` as this is more widely supported. see https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0#non-prefixed-environment-variables